packages/galileo-cdk/src/ai/llms/models/managed-embeddings/custom.asset/code/inference.py

```diff
@@ -0,0 +1,159 @@
+import os
```

Comment:
check: are we using ' or " for python?

Reply:
we don't have linting setup for python yet, so either is fine for now